app/assets/javascripts/preview.coffee

```diff
@@ -104,3 +106,17 @@ $(document).ready ->
     $('.science-keywords-more-item').addClass('is-invisible')
     $('.science-keywords-more-toggle').removeClass('is-invisible')
     $(this).addClass('is-invisible')
+
+  $('.tab-label').on 'click', (e) ->
+    $('.tab-label').removeClass('active')
+    $currentTab = $(this)
+    $currentTab.addClass('active')
+    panelId = $currentTab.attr('for')
+    $('.tab-panel').addClass('is-invisible')
+    $('#' + panelId).removeClass('is-invisible')
+
+    # is-invisible class is not working for the spatial box
```

Comment:
I think we don't need this comment. It is true that `is-invisible` is not working, but that is because of the way the class interacts with the element. More importantly, I also tried using `show()` and `hide()`, which worked, but this way ends up being a cleaner way to do it because of clicking in between tabs.

Reply:
removed